internal/controllers/rest_controller.go

```diff
@@ -42,9 +44,13 @@ func NewRESTController(
 
 // Boot initializes creating some routes.
 func (rc RESTController) Boot(s *server.Server) {
-	s.Router.GET(rc.configuration.AuthServiceURL, gin.WrapF(rc.CheckForAuthentication))
+	s.Router.Use(rc.allowlistMiddleware())
+
+	s.Router.GET("/", gin.WrapF(rc.CheckForAuthentication))
+	s.Router.GET("/:path", gin.WrapF(rc.CheckForAuthentication))
+	s.Router.Any("/:path/*sources", gin.WrapF(rc.CheckForAuthentication))
```

Comment:
why do we need this?

Reply:
it's necessary because this route handles any sources, e.g:

```
/example.js
/apis/laboratory
/assets/img/01.png
/?ns=username
```

if it doesn't exist, we get 404.

Reply:
that's not suppose 🤔

Reply:
that's the gateway doing it